### ISSUE TITLE
Vue router 4 document doesn't support Korean language.

### ISSUE THREAD

**Issue by jsw3363** [vuejs/vue-router-next#1229]
### What problem does this feature solve?
<img width="438" alt="오픈_2-1" src="https://user-images.githubusercontent.com/91763903/144624533-c83d9cfb-459b-4a3b-9480-c2eb420b175c.PNG">

https://router.vuejs.org/
In the "Vue-Router 3" document, there's a Korean page in "Language"
but, in "Vue-Router 4" document, Korean does not exist in "Language".

https://github.com/vuejs-kr/vue-router-next
I know that most of the translation progress of "vue router 4"  has been completed in the github Korean translation project, but there is no Korean in the document.

So, Could you add Korean to the "vue router 4" document?

### What does the proposed API look like?
<img width="528" alt="오픈_2" src="https://user-images.githubusercontent.com/91763903/144655041-18b5f2bd-a49f-4b13-bca0-c731d63145db.PNG">

Like "vue router 3" document, I hope that "vue router 4" will also apply support to Korean Language.

**Comment by posva** [comment]
Can you deploy the Korean docs somewhere so we can link them with an external link in the language nav menu?

@bencodezen is that what you had in mind? I prefer keeping only the English and Chinese docs within the repository.

**Comment by bencodezen** [comment]
> Can you deploy the Korean docs somewhere so we can link them with an external link in the language nav menu?
> 
> @bencodezen is that what you had in mind? I prefer keeping only the English and Chinese docs within the repository.

Yeah I was thinking of just adding a link to an external docs that is not maintained within the main repo. Does that work?

**Comment by posva** [comment]
yes!

**Comment by bencodezen** [comment]
@jsw3363 Is there a link where the Korean docs are deployed? Once we have that, we can add it as a link to the main docs site. Thanks!

**Comment by narusas** [comment]
http://router.vuejs-korea.org 
translation is in progress

**Comment by narusas** [comment]
https://router.vuejs-korea.org/ko/index.html
